resources/codegen_inputs/stat.rs

```diff
@@ -1,16 +1,19 @@
-#![parse_module(tables::layout::stat)]
+#![parse_module(read_fonts::tables::stat)]
 
 /// [STAT](https://docs.microsoft.com/en-us/typography/opentype/spec/stat) (Style Attributes Table)
 table Stat {
     /// Major/minor version number. Set to 1.2 for new fonts.
     #[version]
+    #[compile(MajorMinor::VERSION_1_2)]
     version: MajorMinor,
     /// The size in bytes of each axis record.
+    #[compile(8)]
```

Comment:
What `compile` indicates is that this value cannot be modified by the user, and is instead computed at compile time. The attribute was previously called `compute`, but that also seemed ambiguous.

Maybe this would be clearer if we defined a constant like `AXIS_RECORD_SIZE`, and used that instead of a literal?